build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -200,30 +200,6 @@ Executes /t:Pack on all projects matching src/*/*.csproj.
 
 <!--
 ###################################################################
-Target: PackSharedSources
-
-Creates a content files package for all each directory in
-that matches "$(RepositoryRoot)/shared/*.Sources".
-###################################################################
--->
-  <Target Name="PackSharedSources" Condition="Exists('$(SharedSourcesFolder)') OR '@(SharedSourceDirectories)' != ''">
```

Comment:
@pranavkm Right. I misunderstood what this was. Your link above looks to be what I was after. Although, I'm curious if removing this step speeds up the builds any, I probably won't touch https://github.com/aspnet/Mvc/pull/5905 any more until it is reviewed.